docs/advanced/transaction-details.md

```diff
@@ -2,11 +2,28 @@
 
 ---
 
-Decred transactions are transfers of DCR that exist within blocks. Transactions are comprised primarily of inputs and outputs, though they have a few other fields of data as well. 
+Decred transactions are transfers of DCR that exist within blocks. Transactions are comprised primarily of inputs and outputs, though they have a few other fields of data as well. In this page, we will examine the types of transactions Decred supports, the basic format common to all transactions, and examine an example transaction in detail. 
+
+
+## Transaction Types
+
+To enable Decred's consensus rules, transactions are split into two separate trees within a block: the regular transaction tree and the stake transaction tree. The stake transaction tree contains transactions related to ticket buying and ticket voting rewards. The regular transaction tree contains all other transactions. 
+
+In addition to regular transactions (sending DCR from one address to another), there are a few special transaction types to be aware of:
+
+* **Coinbase transaction:** Every block which is mined contains a single coinbase transaction. A coinbase transaction will typically have three outputs: one output that contains the encoded block height (used to ensure that coinbase hash collisions are impossible), one output that creates the coins added to the project treasury (10% of the block reward), and one output that creates the coins paid to the PoW miner for creating the block (60% of the block reward). Coinbase transactions are part of the regular transaction tree, which means that they will be rejected if Proof-of-Stake (PoS) voters vote to disapprove the block which contains them. Decred created in coinbase transactions cannot be spent until the coinbase maturity period has passed.
+* **Stakebase transaction:** Every block starting at the stake validation height which is mined will contain a stakebase transaction for each ticket which voted on that block. Stakebase transactions are located in the stake transaction tree and have two outputs: the Decred which was spent to purchase the ticket, and the newly created Decred constituting the reward for voting. Stakebase transactions are also created when an expired or missed ticket is revoked, however there will be no voting reward created in these cases. Stakebase transactions belong to the stake transaction tree, which means that these transactions cannot be rejected by PoS voters, even if they vote to disapprove the block which contains them.
```

Comment:
It's not true that stakebase transactions only contain two outputs.  It's really quite complicated and there are a lot of rules around it.  I'm not sure it's really appropriate to call that out here as that seems more suitable for developer docs.

I would say perhaps just:

>Every block starting at the stake validation height which is mined will contain a stakebase transaction for each ticket which voted on that block. Stakebase transactions are located in the stake transaction tree and consist of a very specific form which identifies the ticket that is associated with it as well as produces the Decred which was spent to purchase said ticket and the newly created Decred constituting the reward for voting.  Stakebase transactions are also created when an expired or missed ticket is revoked, however there will be no voting reward created in these cases. Stakebase transactions belong to the stake transaction tree, which means that these transactions cannot be rejected by PoS voters, even if they vote to disapprove the block which contains them.

Reply:
Also, I just noticed there is a repetitive bit in there "Stakebase transactions are located in the stake transaction tree ...." and later "Stakebase transactions belong to the stake transaction tree, which means ...."

Reply:
Reworded for flow:

>Stakebase transaction: Every block starting at the stake validation height which is mined will contain a stakebase transaction for each ticket which voted on that block. The stakebase transaction will identify the ticket that voted, as well as produce the Decred which was spent to purchase the ticket, and the newly created Decred constituting the voting reward. Stakebase transactions belong to the stake transaction tree, which means that these transactions cannot be rejected by PoS voters, even if they vote to disapprove the block which contains them.

This does raise a question I've had for a while though: when you say the stakebase transaction "produces" the Decred which was spent to purchase the ticket, do you mean that it "unlocks" Decred that was locked in a smart contract when the ticket was purchased? Or does it create Decred that was destroyed? I assume the former.

Reply:
I mean that it actually produces new coins.  Under the hood, the act of acquiring a ticket destroys the Decred until either a vote or a revocation mints the same amount in new coins to replace them.  The documentation refers to this process as "locking" because it's conceptually easier to understand and the coins are indeed not spendable until a vote or revocation effectively "unlocks" them.

Reply:
As far as the proposed text, it sounds good and is accurate.

Reply:
Interesting. Ok I think we're good to merge then unless anyone else chimes in.